homeassistant/helpers/config_validation.py

```diff
@@ -219,6 +221,17 @@ def validate(obj):
     return validate
 
 
+def validate_values(valid_values):
```

Comment:
I think this will convert the set back to a list `vol.In(list(SENSOR_TYPES.keys()))`.